README.md

```diff
@@ -28,26 +28,117 @@ The website consumes this repo through a ruby gem. The gem points at the `master
 1. Open PR & ask the website team to review
 1. Follow the deployment process for website (if in doubt, ask the website team for help)
 
+
+
 ## Publishing to JSBin
 
 This configuration is not required for basic documentation creation and modification, so most
-editors can simply skip to
-[Forking and running locally](#forking-and-running-locally).
+editors can simply skip to [Forking and running locally](#forking-and-running-locally).
 
 If you will be creating or modifying code within this repository that is to be uploaded automatically
-to [JSBin](https://jsbin.ably.io/), for our "Try it now code editor",
-then you will need to create yourself a JSBin config file:
+to [JSBin](https://jsbin.ably.io/), for our "Try it now code editor", then you will need to create yourself a JSBin config file:
+
+### Ably staff members
+
+To obtain the API key you need to run this command in the terminal, you must ensure you have installed the hero-cli tools (Instructions on [installing the Heroku toolbelt](https://devcenter.heroku.com/articles/heroku-cli) )
```

Comment:
@fliptopbox you also need to have postgresql installed, which i did not

Reply:
`hero-cli` or `heroku-cli` or `heroku toolbelt`

Reply:
@MarkWoulfeAbly I will add a mention to postgres install guide
@kennethkalmer not sure what your mean. please explain